server/conf/cms/secrets.sample.py

```diff
@@ -57,11 +56,10 @@
 # CMS Site (allows for multiple sites on a single CMS)
 _SITE_ID = 1
 _CMS_TEMPLATES = (
-    # Customize this
-    # FAQ: First template is default template
-    # REF: http://docs.django-cms.org/en/latest/how_to/install.html#templates
+    # Default template with standard and custom (per-project) choices
+    # NOTE: To have per-project styles, the custom default template is required
```

Comment:
The removed comments are removed because the customization is happening… now, in this PR.

The added comments come from CMS's default secrets.